web-test-runner.config.js

```diff
@@ -0,0 +1,13 @@
+import { esbuildPlugin } from "@web/dev-server-esbuild";
+
+export default {
+	nodeResolve: true,
+	plugins: [
+		esbuildPlugin({
+			jsx: true,
+			loaders: {  '.js': 'jsx' },
+			jsxFactory: 'h',
+			jsxFragment: 'Fragment',
```

Comment:
The esbuild plugin doesn't support configuring the automatic transform at this time, hence the pragma.